### DISCUSSION TITLE
Where would I find temporary tables?

### DISCUSSION THREAD

**Discussion by michaelherger** [pawelsalawa/sqlitestudio#4847] [General]
I created a temporary table in SQLiteStudio. I can access it in a query. But it's not listed in the Databases explorer/navigation tree. Is there a way to show all temporary tables easily?

**Comment by tuffnatty** [comment]
There's a feature request, see #3487.

**Comment by michaelherger** [comment]
Thanks! I did try to find something, but didn't go that far back 😬.
